src/backends/backend.js

```diff
@@ -42,7 +43,7 @@ const slugFormatter = (template = "{{slug}}", entryData) => {
     return identifier;
   };
 
-  return template.replace(/\{\{([^\}]+)\}\}/g, (_, field) => {
+  let slug = template.replace(/\{\{([^\}]+)\}\}/g, (_, field) => {
```

Comment:
Although we allow some mutable ops for dense logic or areas where performance is a concern, we generally want to avoid them. I'd recommend something more like:

```
const slug = template.replace(...)
  .method()
  .method();
```

I'd also move `sanitizeFilename` and the trailing character replacement to a new `sanitizeSlug` export from `urlHelper`, so you can just `return sanitizeSlug(slug)`.

Reply:
Done, but I'm not sure if I have it the best way. Could you please review `sanitizeSlug`?